checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -364,6 +364,19 @@ class checkoutCustom {
     $(`.mini-cart .cart-items`).addClass('v-loaded')
   }
 
+  removeCILoader() {
+    let counter = 0
+    const max = 10
+    const cartItems = setInterval(function() {
+      if ($('.cart-items').length > 0 || counter >= max) {
+        clearInterval(cartItems)
+        $(`.cart-items`).addClass('v-loaded')
+      } else {
+        counter++
+      }
+    }, 500)
```

Comment:
How did you test it and reach the max = 10 ?
In addition, I'd like to recommend to you test it by changing your network connection to **slow 3g** and check if the "max = 10" is enough.

<img width="196" alt="image" src="https://user-images.githubusercontent.com/5812946/161833777-12584915-4c81-48cc-8b9b-3067385115f0.png">

Reply:
Hello @arturmagalhaesjr , i decided 10 as max as i tested with the debugger counting manually how many iterations take to be good and a regular number was 4, so 10 i think is good enough, and i just tried with slow 3G and it works either, thank you!